contracts/modules/capital/Ramm.sol

```diff
@@ -746,7 +746,14 @@ contract Ramm is IRamm, MasterAwareV2, ReentrancyGuard {
       return;
     }
 
-    uint128 nxmReserveA = (INITIAL_LIQUIDITY * 1 ether / SPOT_PRICE_A).toUint128();
+    uint capital = pool().getPoolValueInEth();
+    uint supply = tokenController().totalSupply();
+
+    uint bondingCurvePrice = pool().getTokenPrice();
+    uint initialPriceA = bondingCurvePrice + 1 ether * capital * (PRICE_BUFFER_DENOMINATOR + PRICE_BUFFER) / PRICE_BUFFER_DENOMINATOR / supply;
+    uint initialPriceB = 1 ether * capital * (PRICE_BUFFER_DENOMINATOR - PRICE_BUFFER) / PRICE_BUFFER_DENOMINATOR / supply;
```

Comment:
1. When calculating `inithalPriceB`, the `PRICE_BUFFER_DENOMINATOR - PRICE_BUFFER` should be just `PRICE_BUFFER` (see comments on the doc and confirm first).
2. Still pending, but it is possible that the `inithalPriceA` will be different as it's not entirely clear to me we set it to the intended value.
2. Are there any tests that assert that the resulting twap price is the one we set?

Reply:
- In the docs the docs for the internalPriceB it's BV *(1-ORACLE_BUFFER)
- For internalPriceA I changed to be the same as Rei said in the comments